### ISSUE TITLE
Next Release? 

### ISSUE THREAD

**Issue by fkasischke** [pykeio/ort#211]
Hi, 
I have a quick question - when are you planning to release the next version with all the changes since the April 27 release? 

Thanks!

**Comment by decahedron1** [comment]
RC3 will be released probably after #202 and a light internal refactor - maybe another week or two.

In the meantime you can use `ort` as a git dependency:
```toml
[dependencies]
ort = { git = "https://github.com/pykeio/ort.git", rev = "a92dd3022eb169280ce1d37c2a92b94124ce515a" }
```

**Comment by fkasischke** [comment]
Thanks 👍
